src/pool/cli.c

```diff
@@ -593,8 +593,12 @@ dc_pool_connect(tse_task_t *task)
 		bool by_label = (bool)(args->label != NULL);
 
 		/* connect by one: UUID or label */
-		if (by_uuid == by_label)
+		if (by_uuid == by_label) {
+			D_ERROR("specify one: (label %sprovided and "
+				"UUID %sprovided)\n",
```

Comment:
Nit - we've moved from 80 characters to 100 characters per line, and this is an occasion where it would make the code a lot more readable.

Reply:
Ah yes thanks for reminding of that change and opportunity to clean up here. Changed in latest commit on the branch.